cypress/e2e/login.cy.js

```diff
@@ -2,10 +2,48 @@
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('https://the-internet.herokuapp.com/login');
   });
 
-  it('', () => {
-    
+  it('User can login with valid credentials', () => {
+    cy.contains('h2', 'Login Page')
+      .should('exist');
+    cy.get('#username') 
+      .type ('tomsmith');
+    cy.get('#password')
+      .type ('SuperSecretPassword!');
+    cy.contains('.fa', 'Login')
+      .click();
+    cy.contains('h2', 'Secure Area')
+      .should('exist');
+     
+  });
+
+  it('User have an error with invalid credentials', () => {
```

Comment:
```suggestion
  it('User have an error with invalid username', () => {
```
create a test for the invalid password as well